js/widgets/behaviors/graph-camera.ts

```diff
@@ -89,7 +89,11 @@ export class GraphCameraModel extends ZoomBase implements IBehave {
       this.visible = is3d ? this.getVisible3d(zoom) : this.getVisible2d(zoom);
     }
 
-    this.save();
+    try {
+      this.save();
+    } catch {
+      // nothing to do here
```

Comment:
Should we throw a `console.log` here, or not worth to pollute that any further?

Reply:
Maybe we should invite [@JoelStansbury](https://github.com/JoelStansbury) to join `jupyrdf`.

Reply:
right; it was throwing lots of errors when e.g. widgets get detached